Rakefile

```diff
@@ -27,15 +73,26 @@ namespace :codegen do
           else
             curl -OL https://github.com/google/protobuf/releases/download/v#{protoc_legacy_version}/protoc-#{protoc_legacy_version}-linux-x86_64.zip
           fi
-          unzip protoc-#{protoc_legacy_version}*.zip
+          mkdir -p #{toolchain_bin_dir}
+          unzip -o protoc-#{protoc_legacy_version}*.zip
```

Comment:
May not be something to tackle, but these files are unziped into `/tmp` toplevel dir and if an existing directory matches the name (`include`), you won't get a clean state.

Reply:
ooh good point, fixed as it's a one line thing